packages/zoltan2/core/src/algorithms/color/Zoltan2_AlgHybridD1.hpp

```diff
@@ -232,11 +232,13 @@ class AlgDistance1 : public Algorithm<Adapter>
           }
         }
       },recoloringSize(0));
+      Kokkos::fence();
       Kokkos::parallel_for(n_local, KOKKOS_LAMBDA(const int& i){
```

Comment:
Does this parallel_for need to specify ExecutionSpace?
Similarly for other methods (D2, D1-2GL, PD2).